core/services/bulletprooftxmanager/models.go

```diff
@@ -48,7 +49,7 @@ type EthTx struct {
 	// GasLimit on the EthTx is always the conceptual gas limit, which is not
 	// necessarily the same as the on-chain encoded value (i.e. Optimism)
 	GasLimit uint64
-	Error    *string
+	Error    null.String
```

Comment:
minor, and it was already like that, but could be a bit more readable as 'ErrorString' or ErrorNullable' , etc. 'Error' seems to heavily suggest the 'error' type

Reply:
Yup, I see your point. Changing it now though would require renaming the column, which probably isn't worth it.

Reply:
ahh yeah in that case definitely not - cool